todo_rxbloc_app/lib/feature_todos_list/components/delete_success_toast.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:provider/provider.dart';
+import 'package:todos_repository_core/todos_repository_core.dart';
+
+import '../../app_extensions.dart';
+import '../bloc/todos_list_bloc.dart';
+
+class DeleteSuccessToast extends StatelessWidget {
+  const DeleteSuccessToast({super.key, required this.child});
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    return RxBlocListener<TodosListBlocType, TodoEntity>(
+      state: (bloc) => bloc.states.isTodoDeleted,
+      listener: (context, state) {
+        ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
It's not good practice to have closures as part of the build method. Please extract it as a private one

Reply:
I'm not completely sure what this means